src/geoda-proxy.d.ts

```diff
@@ -0,0 +1,1138 @@
+import GeoDaLisa from './geoda-lisa';
+import GeoDaWeights from './geoda-weights';
```

Comment:
import types `GeoDaLisa` and `GeoDaWeights` that have been used in functions like `getDistanceWeights(): GeoDaWeights` instead of `getDistanceWeights(): any`